tools/distributions/debian/debian_java.BUILD

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 The Bazel Authors. All rights reserved.
```

Comment:
General question: should things like these live in our source tree or be part of debian packaging?

Probably @philwo has an opinion on this.

Reply:
I think it's just easier to maintain if we keep it in Bazel repo?

Reply:
Yeah, I'd say let's keep it in our repo for now - Debian / Ubuntu is a distribution we officially support and distributions try to keep their patches for upstream packages minimal. If we have it in our repo, we can at least test that we don't accidentally break it.

Reply:
Sounds reasonable.